src/tests/ftest/daos_racer/parallel.py

```diff
@@ -30,6 +30,7 @@ def test_parallel(self):
         :avocado: tags=all,full_regression
         :avocado: tags=hw,large
         :avocado: tags=io,daosracer,daos_racer_parallel
+        :avocado: tags=daos_racer_parallel
```

Comment:
Duplicate tag. But it should be on the 4th line here
```suggestion
        :avocado: tags=io,daosracer
        :avocado: tags=daos_racer_parallel
```